### PR TITLE
Cassandra exporter implementation

### PR DESCRIPTION
**Pull request by emreyalvac**
**Description:** 
Cassandra Exporter for traces and logs

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17910

### CODE REVIEW

**Comment by emreyalvac** [comment]
I'm working on trace and logs test.

File: exporter/cassandraexporter/helper.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package cassandraexporter
+
+import "go.opentelemetry.io/collector/pdata/pcommon"
+
+func attributesToMap(attributes pcommon.Map) map[string]string {
```
Comment (atoulme) [inline exporter/cassandraexporter/helper.go:5]: I think you can do `asRaw()` to get the map with raw types.

**Review by atoulme — COMMENTED** [review]
Nice work, please see my comments to get it over the line

**Comment by emreyalvac** [comment]
Branch is broken. I will fix it @atoulme

**Comment by emreyalvac** [comment]
@atoulme sorry for branch issue. I will continue from this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18515

**Pull request closed without merging**